src/pages/attractions-facilities.html

```diff
@@ -1,222 +1,400 @@
-        <meta charset="utf-8">
-        <meta name="description" content="Practice web page.">
-        <meta name="author" content="shapeland">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta name="refresh" content="800">
-        <meta name="keywords" content="uol theme park">
-        <link rel="stylesheet" type="text/css" href="../css/main.css">
-        <link rel="stylesheet" type="text/css" href="../css/attractions-facilities.css">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> <!--social favigon-->
-
-</head>
+
+  <head>
+
+    <title>Shapeland - Attractions and Facilities</title>
+    <meta charset="utf-8">
+    <meta name="description" content="Practice web page.">
+    <meta name="author" content="shapeland">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="refresh" content="800">
+    <meta name="keywords" content="uol theme park">
+    <link rel="stylesheet" type="text/css" href="../css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> <!--social favigon-->
+    <link rel="stylesheet" type="text/css" href="../css/attractions-facilities.css">
   
-    <body>
-    
-    <!-- Nav Bar -->
-        <header class="sticky">   
-            
-                <div id="navbar">
-                    <div class="nav-container">
-                        <div class="logo-container"><img src="../public/images/shape_logo_circ.png" class="logo"></div>
-                        <div class="menu-container"><b>Attraction & Facilities</b></div>
-                        <div class="menu-container"><b>Ticket info</b></div>
-                    </div>
+  </head>
+
+  <body>
+
+    <!--Attractions and Facilities page body-->
+    <div>
+
+      <!--header carousel-->
+      <section class="slideshow-container">
+
+        <div class="mySlides fade">
+          <img src="../public/images/header-carousel-1.jpeg">
+        </div>
+        
+        <div class="mySlides fade">
+          <img src="../public/images/header-carousel-2.jpeg">
+        </div>
+        
+        <div class="mySlides fade">
+          <img src="../public/images/header-carousel-3.jpeg">
+        </div>
+        
+        <div class="dot_container">
+          <span class="dot"></span> 
+          <span class="dot"></span> 
+          <span class="dot"></span> 
+        </div>
+        
+        <div class="carousel_label">
+          Attractions & Facilities
+        <div>
+
+      </section>
+
+      <!--page content-->
+      <div class="page_content">
+        <!--rides-->
+        <div class="flex-container">
+
+          <div id="article_1" class="flex-child article-slideshow-container">
+            
+            <div class="article-1-mySlides article-fade">
+              <img src="../public/images/ride-1.jpg" style="width:100%">
```

Comment:
Create a class for your images. Let's try not to use inline styling (eg. `style=...`)